lib/bbb/credentials.rb

```diff
@@ -60,25 +61,34 @@ def tenant_info(tenant, key)
 
       info[key]
     end
-
+    
+    ##
+    # TODO: This new mechanism for tenant_credentials should be discarded when tenant settings are implemented in the brocker (LTI-172).
+    ##
     def fetch_tenant_info(tenant)
-      raise 'Multitenant API not defined' if @multitenant_api_endpoint.nil? || @multitenant_api_secret.nil?
+      tenant_credentials = JSON.parse(Rails.configuration.tenant_credentials)[tenant]
+
+      raise 'Multitenant API not defined' if (@multitenant_api_endpoint.nil? || @multitenant_api_secret.nil?) && tenant_credentials.nil?
```

Comment:
This does not seems to be the right approach. This is forcing the setting of the environment variable. It should only be considered if included, otherwise ignore that logic.

It needs refactoring